ts/src/cryptocom.ts

```diff
@@ -359,6 +370,210 @@ export default class cryptocom extends Exchange {
+            let depositEnabled = undefined;
+            let withdrawEnabled = undefined;
+            let feeFound = undefined;
+            let minWithdrawFound = undefined;
+            const networks = {};
+            let withdrawPrecision = this.safeString (entry, 'withdrawal_decimals');
+            const defaultNetworkId = this.safeString (entry, 'default_network');
+            if (withdrawPrecision === undefined) {
+                // todo: precision info is present only in their "public" endpoint, so for "private" endpoint we add this till they fix their response
+                // for USDT & GUSD precision is 2, for most other coins it's 6,7,8 (only several coins have 4). So, assume 6 as default, to cover most of the cases
+                withdrawPrecision = this.inArray (code, [ 'USDT', 'GUSD' ]) ? '2' : '6';
+            }
+            for (let j = 0; j < networkList.length; j++) {
+                const chainEntry = networkList[j];
+                const networkId = this.safeString2 (chainEntry, 'network_id', 'network');
+                const networkCode = this.networkIdToCode (networkId);
+                const isDefaultNetwork = defaultNetworkId === networkId;
+                const feeNetwork = this.safeNumber2 (chainEntry, 'withdrawal_fee', 'withdrawalFees');
+                let withdraw = this.safeValue (chainEntry, 'withdraw_enabled');
+                if (withdraw === undefined) {
+                    withdraw = this.safeInteger (chainEntry, 'withdraw_open') === 1;
+                }
+                let deposit = this.safeValue (chainEntry, 'deposit_enabled');
+                if (deposit === undefined) {
+                    deposit = this.safeInteger (chainEntry, 'deposit_open') === 1;
+                }
+                const minWithdraw = this.safeNumber2 (chainEntry, 'min_withdrawal_amount', 'minWithdrawalAmount');
+                const isMemoType = this.safeString (chainEntry, 'address_type', '') === 'memo';
+                networks[networkCode] = {
+                    'info': chainEntry,
+                    'id': networkId,
+                    'network': networkCode,
+                    'active': undefined,
+                    'deposit': deposit,
+                    'withdraw': withdraw,
+                    'fee': feeNetwork,
+                    'precision': undefined,
+                    'limits': {
+                        'deposit': {
+                            'min': undefined,
+                            'max': undefined,
+                        },
+                        'withdraw': {
+                            'min': minWithdraw,
+                            'max': undefined,
+                        },
+                    },
+                    'name': this.safeString (chainEntry, 'networkDisplayName'),
+                    'isMemoRequired': (isMemoType ? true : undefined), // because of vague values, don't set false
+                    'isDefaultNetwork': isDefaultNetwork,
```

Comment:
these three fields "name, isMemoRequired, isDefaultNetwork" are new fields added by me, because I think they can be unified. at this moment, we can just add them here and in next times I will expand them across other exchanges too gradually.